dashboard/src/features/authn/reducers.js

```diff
@@ -0,0 +1,54 @@
+import { combineReducers } from 'redux'
+
+const connected = (state = false, action) => {
+  if      (action.type == 'CONNECTED_TO_CORE' ||
+           action.type == 'UPDATE_CORE_INFO') return true
+  else if (action.type == 'CORE_DISCONNECT')  return false
+
+  return state
+}
+
+const authenticationRequired = (state = false, action) => {
+  if (action.type == 'AUTHENTICATION_REQUIRED') return true
+
+  return state
+}
+
+const authenticated = (state = false, action) => {
+  if      (action.type == 'AUTHENTICATION_INVALID') return false
+  else if (action.type == 'UPDATE_CORE_INFO') return true
+
+  return state
+}
+
+const clientToken = (state = '', action) => {
+  if      (action.type == 'SET_CLIENT_TOKEN') return action.token
+  else if (action.type == 'ERROR' &&
+           action.payload.status == 401)      return ''
+
+  return state
+}
+
+const authenticationReady = (state = false, action) => {
+  if (action.type == 'AUTHN_LOADED') return true
```

Comment:
Above there's a property called `authenticationRequired`, which is set by an action called `AUTHENTICATION_REQUIRED`. Here, there's a property called `authenticationReady` that is set by an action called `AUTHN_LOADED`...maybe we can make the action name the same as the property?